src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -3,24 +3,40 @@
             [puppetlabs.trapperkeeper.authorization.rules :as rules]
             [puppetlabs.trapperkeeper.authorization.ring :as ring]
             [puppetlabs.ssl-utils.core :as ssl-utils])
-  (:import  (java.net InetAddress)
-            (clojure.lang IFn)))
+  (:import  (clojure.lang IFn)))
 
+(def is-authentic-key
```

Comment:
Would be nice to have the `is-authentic-key` and `name-key` functions in one place - either here or in `rules.clj`.